code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -87,13 +91,25 @@
 	update_icon()
 	COMPILE_OVERLAYS(src)
 
-/obj/structure/closet/crate/get_door_transform(azimuth_angle, polar_angle, radius_cr)
+/obj/structure/closet/crate/get_door_transform(crateanim_1, crateanim_2)
 		var/matrix/M = matrix()
 		M.Translate(0, -door_hinge)
-		M.Multiply(matrix(1, -sin(polar_angle)*sin(azimuth_angle)* radius_cr, 0, 0, radius_cr*cos(azimuth_angle)*sin(polar_angle), 0))
+		M.Multiply(matrix(1, crateanim_1, 0, 0, crateanim_2, 0))
 		M.Translate(0, door_hinge)
 		return M
```

Comment:
This portion is indented wrongly: there are two tabs

Reply:
No idea how i got that in but i am on it